x/evm/keeper/grpc_query.go

```diff
@@ -76,3 +76,16 @@ func (q Querier) DepositState(c context.Context, req *types.DepositStateRequest)
 
 	return &types.DepositStateResponse{Status: s}, nil
 }
+
+// PendingCommands implements the pending commands query
+func (q Querier) PendingCommands(c context.Context, req *types.PendingCommandsRequest) (*types.PendingCommandsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	ck := q.keeper.ForChain(req.Chain)
```

Comment:
should probably check if the chain exists (`HasChain`) first

Reply:
good point. Should also add this check for the rest of queries at a separate PR.

Reply:
added chain check